src/listener/UserEventListener.php

```diff
@@ -21,25 +21,49 @@
 
 use common_exception_Error;
 use core_kernel_classes_Resource;
+use core_kernel_classes_Property;
 use oat\authKeyValue\AuthKeyValueUserService;
 use oat\authKeyValue\helpers\OntologyDataMigration;
+use oat\generis\model\data\event\ResourceUpdated;
 use oat\oatbox\service\ConfigurableService;
 use oat\tao\model\event\UserRemovedEvent;
 use oat\tao\model\event\UserUpdatedEvent;
+use oat\generis\model\OntologyRdf;
+use oat\tao\model\TaoOntology;
 
 class UserEventListener extends ConfigurableService
 {
     /**
-     * @param UserUpdatedEvent $event
+     * @param UserUpdatedEvent|ResourceUpdated $event
```

Comment:
Note: This docblock line is redundant and can be deleted as the parameter type is already determined by the method definition.